core-tests/shared/src/test/scala/zio/ZIOBaseSpec.scala

```diff
@@ -5,4 +5,36 @@ import zio.test._
 
 trait ZIOBaseSpec extends DefaultRunnableSpec {
   override def aspects = List(TestAspect.timeout(60.seconds))
+
+  sealed trait ZIOTag {
+    val value: String
+  }
+  object ZIOTag {
+    case object Caching extends ZIOTag { override val value = "caching" }
```

Comment:
From the perspective of a user who will be running these, I suggest each tag be 1 word long, maximum. And shortest form of a word, e.g. `ErrorHandling` => `Errors`, `Memoization` => `Memo`, `ResourceHandling` => `Resource`, etc.

Reply:
Yeah, makes sense, thanks for the suggestion @jdegoes!